conan/tools/cmake/cmakedeps/templates/target_configuration.py

```diff
@@ -78,103 +89,132 @@ def template(self):
         set(CMAKE_PREFIX_PATH {{ '${' }}{{ pkg_name }}_BUILD_DIRS{{ config_suffix }}} {{ '${' }}CMAKE_PREFIX_PATH})
 
         {% if not components_names %}
-        ########## GLOBAL TARGET PROPERTIES {{ configuration }} ########################################
```

Comment:
Sorry for this diff, it is mostly tabulation of the CMake code, I think now it is much more readable and the generated files don't look so bad, actually improves readability too.